src/overlays/actors/ovl_Obj_Blockstop/z_obj_blockstop.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_obj_blockstop.c
  * Overlay: ovl_Obj_Blockstop
- * Description:
+ * Description: Snowhead Temple - plays a cutscene when a block falls into a hole
```

Comment:
I mean these two blocks:
![NKZE01_2021-11-20_15-30-45](https://user-images.githubusercontent.com/12245827/142743871-725e4b60-9d5c-40d6-8a4b-18625899b10e.png)
![NKZE01_2021-11-20_15-31-26](https://user-images.githubusercontent.com/12245827/142743873-72973d98-9890-4416-aee4-58324e4e7280.png)
The holes that they fall into have a Blockstop inside them. I can't think of a good way to describe them both, since they're completely unrelated push blocks.

Reply:
oh thats not a switch they hit but this other actor? neat

Reply:
Yeah, I guess you could think of Blockstop as an invisible switch that only the block can interact with. It's hard to convey that concept with a simple description though.

Reply:
Yeah using as few words as possible is an issue here.

I would have probably gone with "Push Block Trigger - Blockstop switch triggered by pushblock (Snowhead)"

Reply:
Well, that's better than what I already have, so let's go with it. It can always be improved later.